cypress/integration/Crosscutting/HttpStatusCodeTest/index.js

```diff
@@ -0,0 +1,11 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('correct status code {string} is received for the {string}', (code, path) => {
+    cy.request({
+        url: path,
+        followRedirect: false, 
```

Comment:
this should not be here, remove followRedirect arg

Reply:
done